dolphin/exception.py

```diff
@@ -302,3 +302,7 @@ class SSHException(DolphinException):
 
 class SSHInjectionThreat(DolphinException):
     message = _("SSH command injection detected: %(command)s")
+
+
+class AlertSourceNotFound(NotFound):
+    message = _("Alert source for storage %(storage_id)s could not be found.")
```

Comment:
```suggestion
    message = _("Alert source for storage %(storage_id) could not be found.")
```

Reply:
This 's' does not means storages but the format of the exception message.

Reply:
Sorry, got it..